### ISSUE TITLE
BioFormats: Unable to read BMP file

### ISSUE THREAD

**Issue by braymp** [CellProfiler/CellProfiler#1352]
Running CP from source, drag/drop the attached BMP into Images and dbl-click to open. Opening fails with the following error:
```
Traceback (most recent call last):
  File "C:\Trunk\CellProfiler\cellprofiler\gui\cpfigure.py", line 1896, in show_image
    image = load_image_url(url)
  File "C:\Python27\lib\site-packages\bioformats\formatreader.py", line 525, in load_using_bioformats_url
    channel_names)
  File "C:\Python27\lib\site-packages\bioformats\formatreader.py", line 793, in read
    image.shape = (height, width, 3)
ValueError: total size of new array must be unchanged
```
These are BMPs that earlier builds were able to process with no problems.

Attachments:
http://cellprofiler.org/issues/uploaded/bsNZ_7/sample7.bmp

**Comment by LeeKamentsky** [comment]
it's a bug in bioformats that's fixed in 1.0.4 - can you update to that and try again?

**Comment by braymp** [comment]
That did it; closing.
